contrib/mesos/pkg/executor/executor.go

```diff
@@ -329,6 +356,43 @@ func (k *KubernetesExecutor) LaunchTask(driver bindings.ExecutorDriver, taskInfo
 	go k.launchTask(driver, taskId, pod)
 }
 
+func (k *KubernetesExecutor) handleChangedApiserverPod(pod *api.Pod) {
+	taskId := pod.Annotations[meta.TaskIdKey]
+	if taskId == "" {
+		log.V(5).Infof("ignoring pod update for %s/%s because %s annotation is missing", pod.Namespace, pod.Name, meta.TaskIdKey)
```

Comment:
can we add a comment here along these lines?

On Mon, Sep 21, 2015 at 1:26 PM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/executor/executor.go
> <https://github.com/kubernetes/kubernetes/pull/13966#discussion_r39998824>
> :
>
> > @@ -329,6 +356,43 @@ func (k *KubernetesExecutor) LaunchTask(driver bindings.ExecutorDriver, taskInfo
> >  	go k.launchTask(driver, taskId, pod)
> >  }
> >
> > +func (k *KubernetesExecutor) handleChangedApiserverPod(pod *api.Pod) {
> > +	taskId := pod.Annotations[meta.TaskIdKey]
> > +	if taskId == "" {
> > +		log.V(5).Infof("ignoring pod update for %s/%s because %s annotation is missing", pod.Namespace, pod.Name, meta.TaskIdKey)
>
> A pod can be assigned to this node, but not scheduled ("pre-scheduled
> pods").
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13966/files#r39998824>.
>